src/assets/events-list-table/events-list-table.django

```diff
@@ -51,7 +51,7 @@
         </td>
         <td class="col-payment">
           {% if payment %}
-            {{ payment.get_payment_method_display }}
+            {{ payment.get_method_display }}
```

Comment:
We can add a method to `Payment` if this needs to be displayed differently that it is stored. I want to keep all the info in `data`.